packages/api/cms-api/src/mikro-orm/migrations/Migration20230302145445.ts

```diff
@@ -2,12 +2,20 @@ import { Migration } from "@mikro-orm/migrations";
 
 export class Migration20230302145445 extends Migration {
     async up(): Promise<void> {
-        this.addSql('alter table "PageTreeNode" add column "updatedAt" timestamp with time zone;');
-        this.addSql('update "PageTreeNode" SET "updatedAt"=CURRENT_DATE');
-        this.addSql('alter table "PageTreeNode" alter column "updatedAt" set not null');
+        this.addSql('alter table if exists "PageTreeNode" add column if not exists "updatedAt" timestamp with time zone;');
+        this.addSql(
+            `do $$
+            begin
+                if exists (select 1 from pg_class where relname='PageTreeNode')
```

Comment:
You could also execute this query before the addSql statements and add an early return. Would be easier to read and understand IMO